projects/plugins/protect/src/js/components/threats-list/index.jsx

```diff
@@ -123,27 +130,32 @@ const ThreatsList = () => {
 											/>
 										</>
 									) }
-									<Button
-										ref={ setDailyAndManualScansPopoverAnchor }
-										variant="secondary"
-										className={ styles[ 'list-header-button' ] }
-										isLoading={ scanIsEnqueuing }
-										onClick={ handleScanClick() }
-									>
-										{ __( 'Scan now', 'jetpack-protect' ) }
-									</Button>
-									<OnboardingPopover
-										id="paid-daily-and-manual-scans"
-										position={ isSm ? 'bottom left' : 'middle left' }
-										anchor={ dailyAndManualScansPopoverAnchor }
-									/>
-								</>
+									<div>
+										<ScanButton ref={ setDailyAndManualScansPopoverAnchor } />
+										<OnboardingPopover
+											id="paid-daily-and-manual-scans"
+											position={ isSm ? 'bottom left' : 'middle left' }
+											anchor={ dailyAndManualScansPopoverAnchor }
+										/>
+									</div>
+								</div>
 							) }
 						</div>
 						{ hasRequiredPlan ? (
 							<>
 								<div ref={ setUnderstandSeverityPopoverAnchor }>
 									<PaidList list={ list } />
+									<Text className={ styles[ 'manual-scan' ] } variant="body-small">
+										{ createInterpolateElement(
+											__(
+												'If you have manually fixed any of the threats listed above, <manualScanLink>you can run a manual scan now</manualScanLink> or wait for Jetpack to scan your site later today.',
+												'jetpack-protect'
+											),
+											{
+												manualScanLink: <ScanButton variant="link" />,
+											}
+										) }
```

Comment:
Now that this uses the new `ScanButton` component, the `isLoading` state is applied when a scan is enqueued and the display is a little awkward:
![Screen Shot on 2024-08-09 at 09-49-36](https://github.com/user-attachments/assets/612e5e31-a7b3-4744-881a-2ae75bb14c95)